svir/metadata.txt

```diff
@@ -23,6 +23,8 @@ email=staff.it@globalquakemodel.org
 
 # Uncomment the following line and add your changelog entries:
 changelog=
+    3.1.1
+    * When loading Ground Motion Fields, the names of the GMPEs are displayed instead of the realization names
```

Comment:
When loading Ground Motion Fields -> When loading Ground Motion Fields for scenario calculations